.configure

```diff
@@ -1,7 +1,7 @@
 {
   "project_name": "WordPress-Android",
-  "branch": "trunk",
-  "pinned_hash": "6af74d8dbe176d58d75509b7ed1c43f844e93ce3",
+  "branch": "update-wpandroid-release-signing-file-locations",
```

Comment:
What's the plan to bring the changes from this branch back into `trunk`? Anything I can do to help?

Reply:
It looks like @JavonDavis already addressed this in 3a253e5eebbcf98f126b10585b882808a650d142. I am not sure why Github was showing this diff at the time of the review 🤷‍♂️

Reply:
Oh, excellent! 👍

Reply:
I should have looked at the list of commits properly 🤦